### PR TITLE
Update the PyCon India 2021 announcement blog

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
* Move the updated dates and structure to the top
* Add a link to the updates blog

CC: @chkumar246 @palnabarun

### CODE REVIEW

File: content/blog/2021/announcing-pycon-india.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ Category: 2021
 Slug: pycon-india-announcement
 Summary: Announcing the return of PyCon India as an online conference for 2021.
 
+** *There have been some updates to the conference structure*
```
Comment (raukadah) [inline content/blog/2021/announcing-pycon-india.md:8]: May be mention here, Earlier was the following dates, Now it is changed to following. so people does not get confused.

Comment (Sc0rpi0n101) [reply content/blog/2021/announcing-pycon-india.md:8]: Wouldn't that just add more dates to the already a little confusing mix?

Comment (anistark) [reply content/blog/2021/announcing-pycon-india.md:8]: Better to be clear about it. We want to be going for clarity.

**Review by anistark — APPROVED** [review]

File: content/blog/2021/announcing-pycon-india.md (hunk 1 above)
Comment (SaptakS) [reply content/blog/2021/announcing-pycon-india.md:8]: I agree on not adding too many dates. People will get confused. But maybe rephrasing to something like "The updated dates for the conference are...". Though I am no content expert, so will leave it on content experts to figure out the actual copy.

Design wise, I would say to put the dates in bold to make them more prominent.

Another thing to note, which is not related to this PR and is probably worth creating an issue for. Seems like the font-size for the list items is much smaller compared to rest of the text, which kind of gives the notion that the details are some secondary information (which I am guessing it is not). I don't expect a change for that here, but maybe a separate PR to update the css.

**Comment by Sc0rpi0n101** [comment]
I've updated the content as per the suggestions. 

Review again please, @raukadah @anistark @SaptakS

File: content/blog/2021/announcing-pycon-india.md [hunk 2]
```diff
@@ -5,6 +5,18 @@ Category: 2021
 Slug: pycon-india-announcement
 Summary: Announcing the return of PyCon India as an online conference for 2021.
 
+** *There have been some updates to the conference structure. The updated dates for the conference are __September 17th to 20th__*
```
Comment (vipulgupta2048) [inline content/blog/2021/announcing-pycon-india.md:8]: @Sc0rpi0n101 What are these `**` for?
Also, would mentioning the year help?

Comment (Sc0rpi0n101) [reply content/blog/2021/announcing-pycon-india.md:8]: This is just to indicate this is a note, like `** Note: `

I don't think mentioning the year would help much, as just above the line is a massive title with PyCon India 2021.

**Pull request merged**